nym-api/src/unstable_routes/models.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright 2024 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: GPL-3.0-only
+
+use cosmwasm_std::{Addr, Coin};
+use nym_topology::NodeId;
+use serde::{Deserialize, Serialize};
+use utoipa::schema;
+
+#[derive(Clone, Debug, Serialize, Deserialize, utoipa::ToSchema, utoipa::ToResponse)]
+pub struct NyxAccountDelegationDetails {
+    pub node_id: NodeId,
+    #[schema(value_type = String)]
+    pub delegated: Coin,
+    pub height: u64,
+    #[schema(value_type = String)]
```

Comment:
`value_type` is not a string, because a `None` will get serialised to `null` rather than `""`. either set it to `Option<String>` or remove the field completely. alternatively change it to `boolean` to indicate whether it's a legacy vested delegation that somebody didn't bother to migrate (and that we should display a big ❗ next to)